torch/distributed/fsdp/_init_utils.py

```diff
@@ -875,7 +879,8 @@ def _get_compute_device(
     rank: int,
 ) -> torch.device:
     """
-    Determines and returns this FSDP instance's compute device. If the module
+    Determines and returns this FSDP instance's compute device. If specified
+    device by 'device_id', then returns that device. Otherwise, If the module
```

Comment:
nit:
```suggestion
    Determines and returns this FSDP instance's compute device. If a device is
    specified by ``device_id``, then returns that device. Otherwise, If the module
```